data/json/items/generic.json

```diff
@@ -2282,6 +2282,19 @@
     "weight": 420,
     "volume": 4
   },
+  {
+    "type": "GENERIC",
+    "id": "tobacco_raw",
+    "symbol": "%",
+    "color": "light_green",
+    "name": "raw tobacco",
+    "name_plural": "handfuls of raw tobacco",
+    "description": "Various parts of tobacco plant, full of nicotine.  They need to be dried to become smokable.",
```

Comment:
add a "looks_like" field here pointing to "tea_raw" or something similar? It would mean that tilesets would display the tea_raw tile if they don't have a tobacco_raw tile.

Reply:
Most likely candidate to have a tile and look similar would be wild herbs I guess.

Reply:
I decided to go for wild herbs. Raw tea doesn't look like fresh plant matter at all, at least in ChestHole tileset.